src/components/Menu/Menu.tsx

```diff
@@ -101,6 +103,32 @@ export const MenuTrigger = forwardRef<
 ))
 MenuTrigger.toString = () => `.${MENU_TRIGGER_CLASS_NAME}`
 
+type MenuButtonProps = ComponentProps<typeof Button>
+const MENU_BUTTON_CLASS_NAME = 'c-menu-button'
+
+export const MenuButton = forwardRef<
+  ElementRef<typeof Trigger>,
+  MenuButtonProps
+>(({ children, ...props }, forwardedRef) => (
+  <Trigger className={MENU_BUTTON_CLASS_NAME} asChild ref={forwardedRef}>
+    <Button {...props}>
+      {children}
+      <ChevronDown
```

Comment:
better to make a separate `styled()` component so this is not remade on every use.